.github/workflows/release.yaml

```
@@ -17,13 +17,13 @@ jobs:
       - name: Set env
         run:  echo "RELEASE_TAG=${GITHUB_REF:10}" >> $GITHUB_ENV
       - name: checkout code
-        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # tag=v5.0.0
+        uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8 # tag=v6.0.1
         with:
           fetch-depth: 0
       - name: Calculate go version
         run: echo "go_version=$(make go-version)" >> $GITHUB_ENV
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:1:41: Double quote to prevent globbing and word splitting

(shellcheck)